MdeModulePkg/MdeModulePkg.ci.yaml

```diff
@@ -91,6 +91,9 @@
 
     ## options defined .pytool/Plugin/LibraryClassCheck
     "LibraryClassCheck": {
+        "IgnoreLibraryClass": [
+            "HwResetSystemLib"          # MU_CHANGE - "Virtual" LibraryClass

```

Comment:
This will prevent the warnings in the build that this library doesn't support the library class, right?

Reply:
It will ensure that each header file found matches a library class in the package DEC file.